extensions/amazon-aws/src/util/index.ts

```diff
@@ -31,6 +31,8 @@ export function resourceToConsoleLink(resourceId: string | undefined, resourceTy
       )}`;
     case "AWS::CloudFormation::Stack":
       return `${AWS_URL_BASE}/cloudformation/home?region=${AWS_REGION}#/stacks/stackinfo?stackId=${resourceId}`;
+    case "AWS::Glue::JobRuns":
+      return `${AWS_URL_BASE}/gluestudio/home?region=${AWS_REGION}#/editor/job/${resourceId}/runs`;
```

Comment:
question (non-blocking): should `gluestudio` be `glue` instead? I see there is a redirect.

Reply:
`gluestudio` should be right. `glue` doesnt work